packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -64,6 +64,13 @@ export enum OdspErrorType {
      * does not match the one at the server.
      */
     epochVersionMismatch = "epochVersionMismatch",
+
+    /**
+     * File not found or access permissions denied error.
+     * This occurs when the user does not have permissions to the file, the file does not exist, or their
+     * permissions are in the process of being granted.
+     */
+    fileNotFoundOrAccessDeniedError = "fileNotFoundOrAccessDeniedError",
```

Comment:
Why do we need this? can we use DriverErrorType.fileNotFoundOrAccessDeniedError instead? We should not duplicate error codes, as that sends a message to hosts that they need to handle it.

Reply:
Ah got it, added in direct reference to DriverErrorType. I had added it in here since we were explicitly handing that condition similarly to epochVersionMismatch, but I can just reuse the existing variable